tails_files/install.sh

```diff
@@ -61,14 +83,19 @@ if $ADMIN; then
   echo "# HidServAuth lines for SecureDrop's authenticated hidden services" | cat - $ANSIBLE/app-ssh-aths $ANSIBLE/mon-ssh-aths $ANSIBLE/app-document-aths > $ADDITIONS
   if [[ -d "$HOMEDIR/.ssh" && ! -f "$HOMEDIR/.ssh/config" ]]; then
     # create SSH host aliases and install them
-    SSHUSER=$(zenity --entry --title="Admin SSH user" --window-icon=$INSTALL_DIR/securedrop_icon.png --text="Enter your username on the App and Monitor server:")
+    # parse prod-specific.yml YAML into Bash variables
+    eval $(parse_yaml $ANSIBLE/prod-specific.yml)
+    if [[ -z $ssh_users ]]; then
+        echo "Error: ssh_users is not defined in prod-specific.yml." 1>&2
+        exit 1
+    fi
     cat > $INSTALL_DIR/ssh_config <<EOL
 Host app
   Hostname $APPSSH
-  User $SSHUSER
+  User $ssh_users
 Host mon
   Hostname $MONSSH
-  User $SSHUSER
+  User $ssh_users
```

Comment:
@ageis Yes, the usernames should be the same on both servers _for a given Ansible run_, but technically any number of usernames can be granted admin privileges on the servers. Unfortunately we don't currently have a great plan for maintaining multiple support for multiple administrators—see #1175. 

@garrettr is right, though: `ssh_users` in the YAML vars file can be either a string or a list, so we must support both use cases.